### PR TITLE
Document behaviour for Sorting Reference Sets in Mx8.

### PR DESCRIPTION
**Pull request by snajahi**
.

### CODE REVIEW

File: content/refguide/ordering-behavior.md [hunk 1]
```diff
@@ -9,25 +9,37 @@ menu_order: 20
...
+
+Sorting the _Product Names_ column will use the underlined values and not the displayed text! These values are the result of `MIN(productName)` for each Order.
+
+## 3 NULL Values Order Behavior {#null-ordering-behavior}
+
+In SQL, `NULL` is a special marker used to indicate that a data value does not exist in the database. If a sort is applied on a column that contains `NULL` values, the decision whether the `NULLs` should come first or last varies per database type.
+
+### 3.1 NULL Order Behavior by Database Engine
+
+#### 3.1.1 HSQLDB
```
Comment (ishan1729) [inline content/refguide/ordering-behavior.md:30]: _3.1.1_ is a **H4** while 3.1.2, 3.1.3 is a **_H3_**. Let's use **H4** for both.

**Review by ishan1729 — CHANGES_REQUESTED** [review]
Few minor comments, otherwise LGTM!

**Review by ishan1729 — APPROVED** [review]
LGTM!

**Review by Adam-Dupaski — APPROVED** [review]

File: content/refguide7/oql-order-by-clause.md [hunk 2]
```diff
@@ -44,5 +44,4 @@ ORDER BY LastName DESC
 This query retrieves all customers and returns the first and last name sorted on the last name, descending.
 
 {{% /alert %}}
-
-**Note:** For default ordering behavior of NULL values refer to [this documentation](ordering-behavior#null-ordering-behavior).
+For detials on the default ordering behavior of NULL values, see [NULL Values Order Behavior](ordering-behavior#null-ordering-behavior) section of *Order by Behavior*.
```
Comment (snajahi) [inline content/refguide7/oql-order-by-clause.md]: There's a small typo, detials should be details

**Pull request merged**